mbuild/packing.py

```diff
@@ -58,22 +58,28 @@ def fill_box(compound, n_compounds, box, overlap=0.2, seed=12345):
         raise IOError(msg)
 
     box = _validate_box(box)
-
-    n_compounds = int(n_compounds)
-    compound_pdb = tempfile.mkstemp(suffix='.pdb')[1]
-    compound.save(compound_pdb, overwrite=True)
-    filled_pdb = tempfile.mkstemp(suffix='.pdb')[1]
+    if type(compound) != list:
```

Comment:
I think `if isinstance(compound, Compound):` might be better here (similar to how this is done in the Polymer class), as this would also catch containers like numpy arrays.